api/src/main/java/org/openmrs/Patient.java

```diff
@@ -91,6 +93,28 @@ public void setPatientId(Integer patientId) {
 	}
 	
 	/**
+	 * It should reflect allergy status maintained by the supporting infrastructure.
```

Comment:
I thought this would be a description of what the method does. Then a separate should annotation below it.
Can you look at the core API and follow the documentation pattern there?